README.md

```diff
@@ -40,24 +44,7 @@ $ pod install
 
 ### Carthage
 
-[Carthage](https://github.com/Carthage/Carthage) is a decentralized dependency manager that builds your dependencies and provides you with binary frameworks.
```

Comment:
we can still keep everything else except the usage section can be delete, The set up with carthage,  pods and SPM will only help us in the future when we plan to use this else where, Can still remain public.

Reply:
Yeah the carthage change here is because the suggested integration method doesn't work for xcframeworks in the current version of carthage.